Cargo.toml

```diff
@@ -27,29 +27,29 @@ crate-type = ["lib"]
 
 [features]
 default = ["curv-kzen/rust-gmp-kzen"]
-cclst = ["class_group"]
+#cclst = ["class_group"]
 
 [dependencies]
 subtle = { version = "2" }
 serde = { version = "1.0", features = ["derive"] }
-zeroize = "1"
-curv-kzen = { version = "0.9", default-features = false }
-centipede = { version = "0.3", default-features = false }
-zk-paillier = { version = "0.4.3", default-features = false }
+zeroize = "^1.5"
+curv-kzen = { version = "0.10.0", default-features = false }
+centipede = { git = "https://github.com/webb-tools/centipede.git", default-features = false }
```

Comment:
@tbraun96 do you know if these deps were updated already to the point where we don't need forks?